frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -83,6 +82,7 @@ export default class AccordionList extends Component {
     alwaysTogglable: PropTypes.bool,
     alwaysExpanded: PropTypes.bool,
     hideSingleSectionTitle: PropTypes.bool,
+    showSpinner: PropTypes.bool,
```

Comment:
I think this should be function right? Since the default prop value is `item => false`

```suggestion
    showSpinner: PropTypes.func,
```